woocommerce-gateway-ebanx/includes/class-wc-ebanx-gateway.php

```diff
@@ -49,6 +49,8 @@ public function __construct()
 		$this->names = $this->get_billing_field_names();
 
 		$this->merchant_currency = strtoupper(get_woocommerce_currency());
+
+		$this->language = $this->getTransactionAddress('country');
```

Comment:
We need to worry about this, because sometimes we don't have access at this time to `getTransactionAddress('country')`, mainly if the user isn't logged. Please, check more about this.